code/game/gamemodes/wizard/raginmages.dm

```diff
@@ -8,7 +8,7 @@
 	var/making_mage = 0
 	var/mages_made = 1
 	var/time_checked = 0
-	var/players_per_mage = 8 // If the admin wants to tweak things or something
+	var/players_per_mage = 6 // If the admin wants to tweak things or something
```

Comment:
I figured this was OK, since normally you'll only get like 4 mages otherwise, which is disappointingly unraging - the wizards only start surging in at 30 minutes, anyways